integral_view/views/system.py

```diff
@@ -0,0 +1,53 @@
+import django
+import django.template
+
+from integral_view.forms import system_date_time_form
+from integralstor_utils.date_time import update_date_time
+
+def input_date_time(request):
```

Comment:
You dont need a separate view for this. You should just put this  into the GET section of update_system_date_time().